stories/webapp/hierarchy/HierarchyFolderRow.js

```diff
@@ -1,20 +1,34 @@
 import React from 'react';
 import { number, oneOfType, node, arrayOf, bool } from 'prop-types';
 import faker from 'faker';
+import { action } from '@storybook/addon-actions';
 import { FolderRow } from '../../../lib';
+import EditableTextWrapper from '../../../lib/EditableTextWrapper';
 
-export const HierarchyFolderRow = ({ childCount, children, open }) => (
-  <FolderRow
-    className="h-margin-bottom-half"
-    metaText={`${childCount} items`}
-    name={faker.commerce.department()}
-    open={open}
-    showToggle
-    style={{ minWidth: '320px' }}
-  >
-    {children}
-  </FolderRow>
-);
+export const HierarchyFolderRow = ({ childCount, children, open }) => {
+  const name = faker.commerce.department();
+
+  return (
+    <FolderRow
+      className="h-margin-bottom-half"
+      metaText={`${childCount} items`}
+      name={
+        <EditableTextWrapper
+          value={name}
+          className="h-margin-clear h-width-100"
```

Comment:
Isnt this already 100% width https://github.com/gathercontent/gather-ui/pull/543/files#diff-09384d7eb14c24294d2ea796c6ea29deR11 ?

Reply:
💯